control-plane/plugin/src/resources/utils.rs

```diff
@@ -22,6 +22,7 @@ lazy_static! {
         "THIN-PROVISIONED",
         "ALLOCATED"
     ];
+    pub static ref SNAPSHOT_HEADERS: Row = row!["ID", "TIMESTAMP", "SIZE", "SOURCE-VOL"];
```

Comment:
low-priority : Should we extend the plugin test to include this?

Reply:
ref:- https://github.com/openebs/mayastor-control-plane/blob/5a6de87c0612a84dd795094ffbc62c42ee1a95e9/control-plane/plugin/src/resources/tests.rs#L261 

This ensures the tabular output is as proper and not broken in terms of indentation.

Reply:
Let me take a look at this once pagination work is done.